docs/conceptual/More-about-how-ROCm-uses-PCIe-Atomics.rst

```diff
@@ -1,156 +1,90 @@
 
-Future bus technology with richer I/O atomics operation Support
+To learn more about the industry standards and specifications of PCIe, see `PCI-SIG Specification <https://pcisig.com/specifications>`_
 
-  * GenZ
+To learn more about PCIe and its capabilities, see the listed white papers:
 
-New PCIe Endpoints with support beyond AMD Ryzen and EPYC CPU; Intel Haswell or newer CPUs
-with PCIe Generation 3.0 support.
+* `Atomic Read Modify Write Primitives by Intel <https://www.intel.es/content/dam/doc/white-paper/atomic-read-modify-write-primitives-i-o-devices-paper.pdf>`_
+* `PCI express 3 Accelerator White paper by Intel <https://www.intel.sg/content/dam/doc/white-paper/pci-express3-accelerator-white-paper.pdf>`_
+* `PCIe Generation 4 Base Specification includes atomic operations <https://astralvx.com/storage/2020/11/PCI_Express_Base_4.0_Rev0.3_February19-2014.pdf>`_
+* `Xilinx PCIe Ultrascale White paper <https://docs.xilinx.com/v/u/8OZSA2V1b1LLU2rRCDVGQw>`_
 
-  * Mellanox Bluefield SOC
-  * Cavium Thunder X2
+Working of PCIe 3.0 atomic operations in ROCm
+-------------------------------------------------
 
-In ROCm, we also take advantage of PCIe ID based ordering technology for P2P when the GPU
-originates two writes to two different targets:
-
-* Write to another GPU memory
-* Write to system memory to indicate transfer complete
-
-They are routed off to different ends of the computer but we want to make sure the write to system
-memory to indicate transfer complete occurs AFTER P2P write to GPU has complete.
-
-BAR memory overview
-----------------------------------------------------------------------------------------------------
-On a Xeon E5 based system in the BIOS we can turn on above 4GB PCIe addressing, if so he need to set
-memory-mapped input/output (MMIO) base address (MMIOH base) and range (MMIO high size) in the BIOS.
-
-In the Supermicro system in the system bios you need to see the following
-
-  * Advanced->PCIe/PCI/PnP configuration-\> Above 4G Decoding = Enabled
-  * Advanced->PCIe/PCI/PnP Configuration-\>MMIOH Base = 512G
-  * Advanced->PCIe/PCI/PnP Configuration-\>MMIO High Size = 256G
-
-When we support Large Bar Capability there is a Large Bar VBIOS which also disable the IO bar.
-
-For GFX9 and Vega10 which have Physical Address up 44 bit and 48 bit Virtual address.
-
-  * BAR0-1 registers: 64bit, prefetchable, GPU memory. 8GB or 16GB depending on Vega10 SKU. Must
-    be placed < 2^44 to support P2P  	access from other Vega10.
-  * BAR2-3 registers: 64bit, prefetchable, Doorbell. Must be placed \< 2^44 to support P2P access from
-    other Vega10.
-  * BAR4 register: Optional, not a boot device.
-  * BAR5 register: 32bit, non-prefetchable, MMIO. Must be placed \< 4GB.
-
-Here is how our base address register (BAR) works on GFX 8 GPUs with 40 bit Physical Address Limit ::
+The PCIe 3.0 atomic operations feature allows atomic transactions to be requested by, routed through, 
```

Comment:
So up until now we haven't discussed this as being a feature. Are we discussing it as a feature of rocm?

Reply:
Removed and rewrote